lib/horde/horde_dynamic_supervisor.ex

```diff
@@ -897,14 +903,26 @@ defmodule Horde.DynamicSupervisor do
   end
 
   defp maybe_restart_child(_, :normal, pid, _child, state) do
+    if state.process_crdt do
```

Comment:
The following three functions are the actual part that makes transient processes work. Everything else in this file is basically just there to make child_ids available in the state.